src/cobra/core/reaction.py

```diff
@@ -62,20 +71,30 @@ class Reaction(Object):
     ----------
     id : string
         The identifier to associate with this reaction
```

Comment:
Please mention the default value for the parameter in the docstring as:
`The identifier to associate with this reaction (default None).`